models/playlist.model.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require("mongoose");
+const { Video } = require("./video.model");
+
+const playlistSchema = new mongoose.Schema(
```

Comment:
You can destructure Schema here from mongoose then you can use it later directly

Reply:
Done